docs/references/teams/get-team-prefs.md

```diff
@@ -0,0 +1 @@
+Get the team preferences by its unique ID.
```

Comment:
We might want to add a reference to user prefs if they prefer the settings to be stored per user and link to that part of the docs and do the same vice versa. This is relevant for all the endpoints. @gewenyu99 wdyt? I feel we should advocate for similar features.

Reply:
This description is pretty short; we can do this.

Something like:

>Get a team's shared preferences by its unique ID. If a preference doesn't need to be shared by all team members, prefer storing them in [user preferences](url).